src/astra_driver.cpp

```diff
@@ -79,6 +79,10 @@ AstraDriver::AstraDriver(rclcpp::node::Node::SharedPtr& n, rclcpp::node::Node::S
   z_scaling_ = 1.0;
   z_offset_mm_ = 0;
 
+  AstraVideoMode color_video_mode{width, height, framerate, PIXEL_FORMAT_RGB888};
+
+  setColorVideoMode(color_video_mode);
```

Comment:
Actually, I think I tried to adjust the depth image mode. In fact I think that's as important if not more important to be able to control as the color image settings for this demo. I hate to ask you to spend more time on it, but could switch this to affecting the depth image rather than the color? I don't think there is a need to support both right now. We can wait until we have automatic command line argument parsing for ROS parameters to do that.